src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -66,7 +66,7 @@ public RuntimeMirror(string varname, int blockDecl, ProtoCore.RuntimeCore runtim
 
                 variableName = varname;
                 blockDeclaration = blockDecl;
-                StackValue svData = deprecateThisMirror.GetValue(variableName, blockDeclaration).DsasmValue;
+                StackValue svData = deprecateThisMirror.GetSymbolValue(variableName);
```

Comment:
In imperative block, all variables that are out of scope will be garbage collected. The only place that I can think of is debugging where the debugger is in the inner scope.